metpy/interpolate/points.py

```diff
@@ -276,7 +276,7 @@ def interpolate_to_points(points, values, xi, interp_type='linear', minimum_neig
                           rbf_smooth=0):
     r"""Interpolate unstructured point data to the given points.
 
-    This function interpolates the given `values` valid at `points` to the points `xi`. This is
+    This function interpolates the given `values` valid at :class:`points` to the points `xi`. This is
```

Comment:
Why add `:class:`? Neither of the other arguments (e.g. `xi`) have them.

Reply:
Great question! For some reason it threw an error about this one but not the others. There isn't a clear rhyme or reason for a lot of these warnings.